src/contracts/interfaces/IRegistryCoordinator.sol

```diff
@@ -23,15 +31,21 @@ interface IRegistryCoordinator {
         // start taskNumber from which the  operator has been registered
         uint32 fromTaskNumber;
         // indicates whether the operator is actively registered for serving the middleware or not
-        Status status;
+        OperatorStatus status;
     }
 
-    /// @notice Returns the bitmap of the quroums the operator is registered for.
-    function operatorIdToQuorumBitmap(bytes32 pubkeyHash) external view returns (uint256);
+    /// @notice Returns the operator struct for the given `operator`
+    function getOperator(address operator) external view returns (Operator memory);
 
-    /// @notice Returns the stored id for the specified `operator`.
+    /// @notice Returns the operatorId for the given `operator`
     function getOperatorId(address operator) external view returns (bytes32);
 
+    /// @notice Returns the quorum bitmap for the given `operatorId` at the given `blockNumber` via the `index`
+    function getQuorumBitmapByOperatorIdAtBlockNumberByIndex(bytes32 operatorId, uint32 blockNumber, uint256 index) external view returns (uint192);
```

Comment:
this reverts if the block number is incorrect, right? should probably add a `dev` tag with this info.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/76/commits/1a690fa781e87e1c7a88da5f6b80e45f566752af